src/rk-events-core.html

```diff
@@ -8,156 +8,167 @@
   <script>
     (function() {
       'use strict';
-
-      Polymer({
-        is: 'rk-events-core',
-
-        properties: {
-          url: {
-            type: String,
-          },
-          username: {
-            type: String,
-          },
-          password: {
-            type: String,
-          },
-          room: {
-            type: String,
-          },
-          handleAs: {
-            type: String,
-            value: 'json'
-          },
-          receivedMessages: {
-            type: Object,
-            notify: true,
-            readOnly: true
-          },
-          _immutableMap: {
-            type: Object,
-            value: null
+      class RkEventsCore extends Polymer.Element {
+      static get is() { return 'rk-events-core'; }
+        static get properties() {
+          return {
+              /* properties metadata */
+            url: {
+              type: String,
+            },
+            username: {
+              type: String,
+            },
+            password: {
+              type: String,
+            },
+            room: {
+              type: String,
+            },
+            handleAs: {
+              type: String,
+              value: 'json'
+            },
+            receivedMessages: {
+              type: Object,
+              notify: true,
+              readOnly: true
+            },
+            _immutableMap: {
+              type: Object,
+              value: null
+            }
           }
-        },
+        }
 
-        addMessages:  function (msgs, instance)  {
-          console.log("CCCCCAALEEDDD");
 
-          if (this._immutableMap) {
-            if (Immutable.is(this._immutableMap, msgs)) {
-              console.log("IMMUTABLE equal;");
-              return;
-            }
-          } else {
-            this.set('_immutableMap', msgs);
-            console.log("IMMUTABLE null");
-            return;
+    addMessages(msgs, instance) {
+      console.log("CCCCCAALEEDDD");
```

Comment:
yes sir for sure, it was pre written that's why i didn't remove it .But now in the latest commit i have done that . Please have a look. I hope i have done it in the right manner (i.e i did changes in my machine then pushed to the respected remote branch ). If i have done something in the wrong way do let me know . :smile: